maquetado/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <h1 class="title">Hello World</h1>
+  <div class="titles">
```

Comment:
I think that you could better this, try with `<h1 class="title">Front-End Tools</h1>` and